components/director/pkg/inputvalidation/validators_cert_subject_mappings.go

```diff
@@ -30,6 +30,8 @@ var SupportedConsumerTypes = map[consumer.Type]bool{
 	consumer.InstanceCreator:                    true,
 	consumer.FormationViewer:                    true,
 	consumer.TechnicalClient:                    true,
+	consumer.ExternalCertificate:                true,
```

Comment:
Shouldn't we update the director version in the hydrator's `go.mod`, because during the validation in the hydrator we import the `/director/pkg/inputvalidation` and use this map